components/tag-list/tag-list.js

```diff
@@ -65,19 +64,26 @@ class TagList extends LocalizeCoreElement(ArrowKeysMixin(LitElement)) {
 		/** @ignore */
 		this.arrowKeysDirection = 'leftrightupdown';
 		this._chompIndex = 10000;
-		this._items = [];
+		this._hasResized = false;
 		this._resizeObserver = null;
 		this._showHiddenTags = false;
 	}
 
 	disconnectedCallback() {
 		super.disconnectedCallback();
 		if (this._resizeObserver) this._resizeObserver.disconnect();
+		if (this._subtleButtonResizeObserver) this._subtleButtonResizeObserver.disconnect();
 	}
 
 	firstUpdated(changedProperties) {
 		super.firstUpdated(changedProperties);
 
+		const subtleButton  = this.shadowRoot.querySelector('.d2l-tag-list-hidden-button');
```

Comment:
This seems to be working ok, measuring just the once when it's first rendered.  

If anyone in the future decides to actually update the hidden button with a count, they might get themselves into a 🐔 & 🥚  situation since the chomping depends on this, and this would depend on the chomping.